### ISSUE TITLE
Processor limit?

### ISSUE THREAD

**Issue by dustyny** [alex-sherman/deco#52]
I've been testing Deco on a AWS instance with 36 cores and but Deco seems to max out at 16-18. Is there a limit to how many processes Deco can run? 

Thanks 
<img width="1015" alt="screen shot 2018-02-14 at 3 38 59 pm" src="https://user-images.githubusercontent.com/7416333/36229567-50508578-119d-11e8-8b4c-8c65ee099c63.png">

**Comment by dustyny** [comment]
This was my mistake.. I was pointed to the wrong endpoint on my database. I was hitting one server, instead of the cluster.

**Comment by dustyny** [comment]
Sorry my mistake.. The additional processes came up when my Database cluster scaled up

**Comment by alex-sherman** [comment]
Do you mean this is still an issue, or were you able to resolve it?

`muliprocessing.Pool` is supposed to default to the number of physical cores the machine has, but you can explicitly pass in a number of processes to deco:

```python
@concurrent(processes=36)
def do_work():
   ...
```
You may also be hitting some other bottleneck in either deco, or `multiprocessing.Pool`. I've never attempted to run deco on anything with 36 cores myself, so seeing someone trying it is awesome!

**Comment by dustyny** [comment]
Thank you for getting back to me so quickly. I've had to manually set the number of CPUs (I can try it again and see if that is true. 

I didn't realize I was pointed to the wrong end point (write vs read) and my Database wasn't scaling up. Since the database server was maxed out at 100%, the other Deco processes were sitting idle. Once I correct that, the cluster scaled up and I was able to use all 36 processors.. 

Great work btw.. I've never had such an easy time scaling up python.. 👍
